src/features/P0_create_package_catalog.py

```diff
@@ -30,18 +28,13 @@ def walklevel(some_dir, level=1):
             del dirs[:]
 
 
-def run():
-    parser = setup_command_line()
-    args = parser.parse_args()
-    logging = setup_log_level(args)
-
+def create_package_catalog(path, date_now, logging, lts):
     _tmp_pkg_tuple_dirs = []
-    logging.info("Processing package index with root {path}".format(path=args.index))
-    for _, pkg_tuple, _ in walklevel(args.index, level=0):
+    logging.info("Processing package index with root {path}".format(path=path))

```

Comment:
Si pones una `f"text"` antes del texto que quieres logear te permite agregar variables entre `{variable}`

```suggestion
    logging.info(f"Processing package index with root {path}")
```